### PR TITLE
Use glob rather than regex

### PR DESCRIPTION
**Pull request by chitoku-k**
Use globbing and also unquote string literals in `[[ ... ]]` expressions.

### CODE REVIEW

**Review by hexium310 — COMMENTED** [review]
In git.zsh, there are useless regex and quoting on the 388th line.
```zsh
[[ $@ =~ '--multiple' ]]
```
Could you replace this with `_fzf_complete_parse_option` as follows?
```zsh
_fzf_complete_parse_option '' '--multiple' '' "${arguments[@]}" > /dev/null
```

File: src/completers/kubectl.zsh [hunk 1]
```diff
@@ -428,7 +428,7 @@ _fzf_complete_kubectl() {
             return
         fi
 
-        if [[ ${subcommands[1]} = 'get' ]] && [[ $completing_option =~ '^(-L|--label-columns)$' ]]; then
+        if [[ ${subcommands[1]} = get ]] && [[ $completing_option = (-L|--label-columns) ]]; then
             if [[ $prefix =~ ',[^,]*$' ]]; then
```
Comment (hexium310) [inline src/completers/kubectl.zsh:432]:
```suggestion
            if [[ $prefix = *,[^,]# ]]; then
```

Comment (hexium310) [reply src/completers/kubectl.zsh:432]: There are similar to this also on the 782nd, 788th and 794th lines.

**Comment by chitoku-k** [comment]
@hexium310 It looks like `[[ -z $repository ]]` on that line does not make any sense as well. Could it be removed?

**Comment by hexium310** [comment]
> @hexium310 It looks like `[[ -z $repository ]]` on that line does not make any sense as well. Could it be removed?

Maybe.

**Review by hexium310 — APPROVED** [review]

**Pull request merged**